test/unit/contentChunk.js

```diff
@@ -13,49 +14,46 @@ describe('contentChunk: ', () => {
     db = client.db()
   });
 
-  after(function() {
-    MongoClient.close()
+  after(async () => {
+    await db.collection('fs.files').deleteMany({})
```

Comment:
Im not sure moving the deletion to the after makes complete sense. This assumes that the database is in a clean state when starting the tests. It also means that the database doesnt get cleaned on a per test basic but rather as a test suite.